src/commands/mod.rs

```diff
@@ -462,240 +447,63 @@ impl DaemonControl {
         }
         let mut db_conn = self.db.connection();
 
-        // Create transaction with no inputs and no outputs.
-        let mut tx = bitcoin::Transaction {
-            version: 2,
-            lock_time: absolute::LockTime::Blocks(absolute::Height::ZERO), // TODO: randomized anti fee sniping
-            input: Vec::with_capacity(coins_outpoints.len()), // Will be zero capacity for coin selection.
-            output: Vec::with_capacity(destinations.len()),
-        };
-        // Add the destinations outputs to the transaction and PSBT. At the same time
+        // Check the destination addresses are valid for the network and
         // sanity check each output's value.
-        let mut psbt_outs = Vec::with_capacity(destinations.len());
+        let mut destinations_checked: HashMap<bitcoin::Address, bitcoin::Amount> =
```

Comment:
No need for the type declaration here

Reply:
I've removed this and a similar one in `rbf_psbt()`.